rdr_service/dao/ghost_check_dao.py

```diff
@@ -0,0 +1,54 @@
+from datetime import date, datetime
+from typing import Optional
+
+from sqlalchemy.orm import Session
+
+from rdr_service.clock import CLOCK
+from rdr_service.dao.base_dao import BaseDao
+from rdr_service.model.ghost_api_check import GhostApiCheck, GhostFlagModification
+from rdr_service.model.participant import Participant
+
+
+class GhostCheckDao(BaseDao):
+    @classmethod
```

Comment:
I am all for the use of classmethods in the logic, but in keeping with the rest of the design patterns for the daos, is there a reason why we are deviating from that? As well as passing Session as a param instead of it being initialized with the dao classes?

Reply:
We've seen strange cases with the DAOs creating sessions that connect to the local database while running tools against prod or stable. This helps make sure we don't run into that.

Reply:
👍 Understood!